### PR TITLE
docs: Pause method comment rectified

### PR DESCRIPTION
**Pull request by Hitesh0413**
pause method will not resume, it will pause or stop on the current position

# Description

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:`, `chore:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation and added dartdoc comments with `///`, where necessary.
- [x] I have updated/added relevant examples in [example].

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

## Related Issues

[issue database]: https://github.com/bluefireteam/audioplayers/issues
[Contributor Guide]: https://github.com/bluefireteam/audioplayers/blob/main/contributing.md#feature-requests--prs
[Conventional Commit]: https://conventionalcommits.org
[example]: https://github.com/bluefireteam/audioplayers/tree/main/packages/audioplayers/example

### CODE REVIEW

**Review by renancaraujo — APPROVED** [review]

**Comment by Gustl22** [comment]
I don't agree. It's already described one line above. It only wants to clarify what happens, if you resume the stream.
See also the `player.stop()` description.
It it's not clear, we should remove the comment from `player.pause` and `player.stop`

**Comment by Hitesh0413** [comment]
> I don't agree. It's already described one line above. It only wants to clarify what happens, if you resume the stream.
> See also the `player.stop()` description.
> It it's not clear, we should remove the comment from `player.pause` and `player.stop`

Yeah, I just rectified the comment. We can entirely remove that also.

**Comment by spydon** [comment]
I think it would be better to remove this comment and explain in a paragraph about what happens if you call `pause` on an already `paused` game.

**Comment by Hitesh0413** [comment]
> I think it would be better to remove this comment and explain in a paragraph about what happens if you call `pause` on an already `paused` game.

Please decline this PR . I will raise another for this. Thanks

**Comment by Gustl22** [comment]
You can just force-push. Or close it by yourself, like you want :)

**Comment by Hitesh0413** [comment]
> You can just force-push. Or close it by yourself, like you want :)

update please review

File: getting_started.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Particularly on Android, the media player is quite resource-intensive, and this
 
 ### resume
 
-Starts playback from current position (by default, from the start).
+Starts playback from last paused position or from start by default.
```
Comment (spydon) [inline getting_started.md:82]: I don't think this sentence is clearer than the previous one. If it has been stopped after the pause call I assume that resume would play from the beginning, and not from where it was last paused.

I think where the clarification is needed is under the `pause` section, and explain what happens if that method is called twice.

Comment (Gustl22) [reply getting_started.md:82]: Only my opinion:
Calling pause a second time does nothing, as it's already paused. Not quite sure, if we have to state that explicitly (?). I would be fine with just removing the comments at `pause` and `stop`.

It may can be refined for stop like `and also resets the position to the start.`

And on the resume description, it's resuming from the current position, I think that's quite clear, too...

To summarize this: I think it's confusing to write `// will resume where left off` after the `pause` call. As it's not what it does. That's the task of `resume` then.

Comment (spydon) [reply getting_started.md:82]: Agree, so the only thing that should be changed is removing the comments for the code then.

Comment (Hitesh0413) [reply getting_started.md]: done

**Review by spydon — APPROVED** [review]
